mailbox/lucene/src/main/resources/META-INF/spring/mailbox-index-lucene.xml

```diff
@@ -31,10 +31,10 @@
         <constructor-arg index="0" ref="luceneIndex"/>
     </bean>
     <bean id="luceneIndex" class="org.apache.james.mailbox.lucene.search.LuceneMessageSearchIndex">
-        <constructor-arg index="0" ref="maildir-sessionMapperFactory"/>
+        <constructor-arg index="0" ref="jpa-sessionMapperFactory"/>

```

Comment:
The messageMapperFactory alias is defined by MailboxConfigurationBeanFactoryPostProcessor : 

```

                messageMapperFactory = "jpa-sessionMapperFactory";
...

            registry.registerAlias(messageMapperFactory, "messageMapperFactory");
```

Please use messageMapperFactory instead of jpa-sessionMapperFactory